### PR TITLE
Created sbom_generation.yaml file

### PR DESCRIPTION
**Pull request by shivam71**
SBOM generation file created

### CODE REVIEW

File: sbom_generation.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
```
Comment (sid-srini) [inline sbom_generation.yaml:1]: Do update the copyright year to 2023-2024

**Review by sid-srini — CHANGES_REQUESTED** [review]
Overall looks fine. Thanks. Minor change needed just to update the copyright year.

File: sbom_generation.yaml (hunk 1 above)
Comment (shivam71) [reply sbom_generation.yaml]: Updated the commit

**Review by sid-srini — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**